x-pack/filebeat/module/suricata/eve/ingest/pipeline.json

```diff
@@ -40,6 +40,11 @@
       ,"ignore_missing": true
       }
     }
+  , { "lowercase":
+      { "field": "http.request.method"
```

Comment:
You can get rid of the `convert` block above this, if you lowercase from `suricata.eve.http.http_method` with `target_field: http.request.method`